core/identity-hub/src/main/java/org/eclipse/edc/identityhub/processor/data/JwtVerifiableCredential.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.identityhub.processor.data;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.SignedJWT;
+import org.eclipse.edc.identityhub.spi.credentials.model.VerifiableCredential;
+import org.eclipse.edc.identityhub.spi.processor.data.DataValidator;
+import org.eclipse.edc.spi.EdcException;
+import org.eclipse.edc.spi.result.Result;
+
+import java.util.Optional;
+
+public class JwtVerifiableCredential implements DataValidator {
+
+    public static final String DATA_FORMAT = "application/vc+jwt";
```

Comment:
I know it is not straightforward due to the client/identityhub split in this repo, but could we consider centralizing these VC data formats into a dedicated file in the `spi` folder?

Reply:
Yeah we can definitely, i would do it in a separated PR though.